### PR TITLE
SOE-3211: Adding feature change and css

### PR DESCRIPTION
**Pull request by minorwm**
# READY FOR REVIEW

# Summary
See https://stanfordits.atlassian.net/browse/SOE-3211
- This changes the authoring experience for the top banner (formerly the featured image) on the Stanford Page.
- This add the CTA button to support the "Make a Gift" button. See:
https://stanfordits.atlassian.net/browse/SOE-3158

# Needed By (Date)
- Sprint end

# Criticality
- Client is eagerly awaiting this for Giving pages

# Steps to Test
- Switch to this branch: `7.x-2.x-soe-3211-top-banner-styling`
- Clear cache so Drupal can find the module: `drush cc all`
- Revert feature: `drush fr stanford_soe_helper_page`
- Create a new Stanford Page at: `node/add/stanford-page`
- Open the **Add/Edit Top Banner**
- Verify groupings as defined in [SOE-3211](https://stanfordits.atlassian.net/browse/SOE-3211)
NOTE: If **body** and **Call to Action button** are reversed, go to `admin/structure/types/manage/stanford_page/form`, edit (move the **Call to Action** button around), and save.

Also: Because the **Accent Color** is a special case and shared, please set the **Default Value** to **turquoise** at:
`admin/structure/types/manage/stanford-page/fields/field_s_top_banner_color`

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
QA: https://stanfordits.atlassian.net/browse/SOE-3212
Work: https://stanfordits.atlassian.net/browse/SOE-3211 
Styling the CTA: https://stanfordits.atlassian.net/browse/SOE-3158

## More Information
The styling ticket https://stanfordits.atlassian.net/browse/SOE-3158 will depend on this PR.

## Folks to notify
@boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by boznik** [comment]
while loading image i received:
Failed to gather css for the WYSIWYG editor. Please visit the home page and try again.

**Comment by boznik** [comment]
is this supposed to be 'Call to Action Button' or 'Make a Gift Button' ?
https://stanfordits.atlassian.net/browse/SOE-3211

**Comment by boznik** [comment]
if they add banner text without an image, it goes behind the title

**Comment by boznik** [comment]
ideally there should be a depency on the image field for the fields that rely on it, but that can be added later.

**Comment by boznik** [comment]
remind people to clear cache when making banner edits.

**Review by boznik — APPROVED** [review]
ok, this is working as expected. would be nice to make fields that are dependent on the image field, uneditable if there isn't an image.

File: modules/stanford_soe_helper_page/stanford_soe_helper_page.module [hunk 1]
```diff
@@ -15,7 +15,7 @@ function stanford_soe_helper_page_form_alter(&$form, &$form_state, $form_id) {
   if ($form_id == "stanford_page_node_form") {
     $form['field_s_image_info'][LANGUAGE_NONE][0]['#groups']['group_s_image']->label =
       'Add/Edit Image and Text';
-    $form['field_s_top_banner_color'][LANGUAGE_NONE]['#description'] =
+    $form['field_s_top_banner_color']['LANGUAGE_NONE']['#description'] =
```
Comment (sherakama) [inline modules/stanford_soe_helper_page/stanford_soe_helper_page.module:18]: @boznik @minorwm 
LANGUAGE_NONE is a constant, not a string. This should be without quotes.

Comment (sherakama) [reply modules/stanford_soe_helper_page/stanford_soe_helper_page.module:18]: @boznik or @minorwm please confirm that one of you has seen this and are planning to fix it.

Comment (boznik) [reply modules/stanford_soe_helper_page/stanford_soe_helper_page.module:18]: ouch, i thought that had been done.

**Pull request merged**